clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/config/EurekaProviderConfiguration.groovy

```diff
@@ -38,12 +53,56 @@ import java.util.regex.Pattern
 @ConditionalOnProperty('eureka.provider.enabled')
 @ComponentScan(["com.netflix.spinnaker.clouddriver.eureka"])
 class EurekaProviderConfiguration {
+
+  @Autowired
+  Registry registry
+
+  @Autowired
+  ConversionService conversionService
+
+  @Autowired
+  ConfigurableEnvironment environment
+
   @Bean
   @ConfigurationProperties("eureka.provider")
   EurekaAccountConfigurationProperties eurekaConfigurationProperties() {
     new EurekaAccountConfigurationProperties()
   }
 
+  private OkHttpClientConfigurationProperties eurekaClientConfig() {
```

Comment:
This feels wrong-but-right - I don't want to publish a different OkHttpClientConfigurationProperties bean into the context so this is duplicating the functionality of `@ConfigurationProperties`